backend/src/Designer/Services/Implementation/PreviewService.cs

```diff
@@ -34,8 +35,27 @@ public async Task<Instance> GetMockInstance(string org, string app, string devel
         Application applicationMetadata = await altinnAppGitRepository.GetApplicationMetadata(cancellationToken);
         DataType dataType = await GetDataTypeForLayoutSetName(org, app, developer, layoutSetName, cancellationToken);
         string task = await GetTaskForLayoutSetName(org, app, developer, layoutSetName, cancellationToken);
+        bool processShouldActAsReceipt = task == "CustomReceipt";
```

Comment:
By rewriting the variable name it is easier to catch that this is a boolean without reading the type. 😊 

```suggestion
        bool shouldProcessActAsReceipt = task == "CustomReceipt";
```